apiary-metastore-listener/README.md

```diff
@@ -83,8 +83,8 @@ The following shows an example JSON message representing an "ADD_PARTITION" even
       "eventType": "ADD_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
-      "partitionKeys": {"Column_2":"String", "Column_1":"String", "Column_3":"String"},
-      "partitionValues": ["val_1", "val_2", "val_3"],
+      "partitionKeys": {"column_1":"String", "column_2":"Integer", "column_3":"String"},
```

Comment:
Can we lower case the type names so we don't confuse them with Java types? Also, I think Hive uses "int" not "integer".

Reply:
Applies to all the examples below.

Reply:
To be consistent with our layout there should be a space after the : inside the map.